Week1/cash.c

```diff
@@ -0,0 +1,25 @@
+#include <cs50.h>
+#include <stdio.h>
+
+void cashback();
+
+int main(void){
+    float owe = get_float("Change owed: ");
+    int penny = (int)(owe*100);
+    if (penny>0){
+        cashback(penny);
+    }else{
```

Comment:
Please take care for readability; adding spaces is sometime a very good idea.

```suggestion
    } else {
```